app/javascript/components/rooms/RoomCard.jsx

```diff
@@ -60,7 +63,7 @@ RoomCard.propTypes = {
     id: PropTypes.string.isRequired,
     friendly_id: PropTypes.string.isRequired,
     name: PropTypes.string.isRequired,
-    created_at: PropTypes.string.isRequired,
+    last_session: PropTypes.string.isRequired,
```

Comment:
With the conditional rendering on the serializer, I suspect that this will throw prop types warnings when there's no `last_session`,

Reply:
Good catch, done